cabal-install/src/Distribution/Client/CmdStatus.hs

```diff
@@ -0,0 +1,367 @@
+{-# LANGUAGE CPP #-}
+{-# LANGUAGE RecordWildCards #-}
+{-# LANGUAGE NamedFieldPuns #-}
+{-# LANGUAGE LambdaCase #-}
+-----------------------------------------------------------------------------
+-- |
+-- Module      :  Distribution.Client.CmdIde
+-- Maintainer  :  cabal-devel@haskell.org
+-- Portability :  portable
```

Comment:
Unsure if this means in terms of OS or in terms of Compiler

Reply:
It is copy-pasted, I have no idea what it means :D

Reply:
https://mail.haskell.org/pipermail/haskell/2004-March/013836.html

it appears this is an undocumented/useless haddock feature